dev/bootstrap.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+cd "$DIR/.."
+
+log() {
+  echo "$(basename ${BASH_SOURCE[0]}): $@"
+}
+
+install_hooks() {
+  git config core.hooksPath \
+    || git config core.hooksPath ./dev/hooks
+}
+
+install_rust_nightly() {
+  rustup toolchain install nightly
```

Comment:
why are we installing nightly?

Reply:
so that uses can run the formatter, otherwise it will fail to run in the hook.